reports/2025-07-21-openscapes-hub-pods-dying

```diff
@@ -0,0 +1,76 @@
+# Incident report July 21 2025 - Openscapes hub pods dying post-stress-testing
+ 
+ 
+## Summary
+ 
+On July 21, 2025, we received a support request from Openscapes asking for confirmation that their hub would be resilient throughout a large, 700 attendee, workshop session the next day. 
+That evening Yuvi completed a stress test, adding thousands of named servers to the database. Early the following morning, alerts fire when the Openscapes workshop hub fails a health check, with hub pod showing OOMKilled status. 
+
+## Resolution
+
+We all theorised that the hub pod dying was related to the stress testing Yuvi did the day before
+Angus noted that the hub database was only 1.6 MB, so we didn't think the issues came from the database being too big
+
+We still thought that the hub database may be corrupted somehow, so we decided to create a backup of the database and try and restart the hub pod without a database
+Restarting the hub without a database worked. We left that in place for the workshop hub, since there is no need for persistent user state in this use case
+
```

Comment:
```suggestion
Restarting the hub without a database worked. We left that in place for the workshop hub, since there is no need for persistent user state in this use case

An [upstream contribution](https://github.com/jupyterhub/jupyterhub/pull/5109) to JupyterHub was made to address inefficient O(N^2) database table join operations.
```